wingbox.py

```diff
@@ -86,51 +87,64 @@ def setup(self, surface, surfacetype):
                        # Aspect ratio definition
                        AR == b**2/S,
 
-                       # Upper bound on maximum thickness
-                       tau <= 0.14,
-
                        # Root stiffness (see Hoburg 2014)
                        # Assumes rh = 0.75, so that rms box height = ~0.92*tmax
                        TCS([0.92*wwb*tau*tcap**2 + Icap <= 0.92**2/2*wwb*tau**2*tcap]),
 
-                       # Stress limit
-                       # Assumes bending stress carried by caps (Icap >> Iweb)
-                       TCS([8 >= Nlift*Mr*AR*q**2*tau/(S*Icap*sigmax)]),
-
-
                        # Posynomial approximation of nu=(1+lam+lam^2)/(1+lam)^2
                        nu**3.94 >= 0.86*p**(-2.38)+ 0.14*p**0.56, # Woody's fit
 
                        # Weight of spar caps and shear webs
-                       Wcap >= 8*rhocap*g*wwb*tcap*S**1.5*nu/(3*AR**0.5),
                        Wweb >= 8*rhoweb*g*rh*tau*tweb*S**1.5*nu/(3*AR**0.5),
                       ]
 
 
         if surfacetype == "wing":
             constraints += [
+                    # Upper bound on maximum thickness
+                    tau <= tau_max,
                     Wstruct >= (Wweb + Wcap),
                     # Shear web sizing
                     # Assumes all shear loads are carried by web and rh=0.75
-                    TCS([12 >= AR*Lmax*Nlift*q**2/(tau*S*tweb*sigmaxshear)]),
+                    TCS([12 >= AR*Lmax*q**2/(tau*S*tweb*sigmaxshear)]),
+                    # Weight of spar caps and shear webs
+                    Wcap >= 8*rhocap*g*wwb*tcap*S**1.5*nu/(3*AR**0.5),
+                    Nlift == 3,
+                    # Stress limit
+                    # Assumes bending stress carried by caps (Icap >> Iweb)
+                    TCS([8 >= Mr*AR*q**2*tau/(S*Icap*sigmax)]),
```

Comment:
@pgkirsch there were some minor errors with Nlift terms that required defining different constraints for wing and tail. I'll take a look later at ways to reduce the number of code lines but I was in a time crunch and wanted to push the correct code before I had to leave my computer